lib/ui/hooks.dart

```diff
@@ -44,14 +84,104 @@ void _updateDisplays(
   PlatformDispatcher.instance._updateDisplays(displays);
 }
 
-@pragma('vm:entry-point')
-void _sendViewConfigurations(List<int> viewIds) {
-  PlatformDispatcher.instance._onSentViewConfigurations(viewIds);
+// TODO(loicsharma): Should we remove this?
+// @pragma('vm:entry-point')
+// void _sendViewConfigurations(List<int> viewIds) {
+//   PlatformDispatcher.instance._onSentViewConfigurations(viewIds);
+// }
+
```

Comment:
```suggestion
// TODO(loicsharma): Should we remove this?
// @pragma('vm:entry-point')
// void _sendViewConfigurations(List<int> viewIds) {
//   PlatformDispatcher.instance._onSentViewConfigurations(viewIds);
// }

```

Reply:
I will remove these, they're not necessary!

Reply:
Removed